lib/iris/tests/unit/fileformats/cf/test_CFGroup.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Unit tests for the :class:`iris.fileformats.cf.CFGroup` class."""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+from iris.fileformats.cf import (
+    CFGroup,
+    CFAuxiliaryCoordinateVariable,
+    CFCoordinateVariable,
+    CFDataVariable,
+)
+
+MagicMock = tests.mock.MagicMock
```

Comment:
I don't plan to change this - I'd much rather re-use something that's already been imported, rather than re-import it. If I'm missing some understanding just let me know.

Reply:
I now understand that the access within `iris.tests` is a legacy item, and it would be better practice to import from its original source (now its in the standard library).

Reply:
c499ab5535274a7e125794c69803d7d893cc578f